### ISSUE TITLE
Spamming with two players on the server

### ISSUE THREAD

**Issue by Lerphie** [PluginBugs/Issues-CosmeticsCore#122]
### Terms

- [X] I'm using the very latest version of CosmeticsCore and its dependencies.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-CosmeticsCore/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://lonedev.gitbook.io/cosmeticscore/) to know if a solution is already known.
- [X] I already searched on the **Discord** server to know if anyone already provided a solution in the past for the issue.

### Discord tag (optional)

Coder#6926

### What happened?

I have a server spamming with two players on the server. If one of them logs out of the server, the spam disappears

### Steps to reproduce the issue

?

### Server version

This server is running Purpur version git-Purpur-1171 (MC: 1.16.5) (Implementing API version 1.16.5-R0.1-SNAPSHOT)

### CosmeticsCore Version

![image](https://user-images.githubusercontent.com/66962762/231610978-5efd542c-0c21-4a46-aa85-0743b8df3413.png)

### ProtocolLib Version

![image](https://user-images.githubusercontent.com/66962762/231610998-7e0ff1dc-cff0-4bb7-b65c-6333ca22c1f2.png)

### LoneLibs Version

![image](https://user-images.githubusercontent.com/66962762/231611015-0c04fcf6-ee8d-417c-8137-bf360209baf0.png)

### Error (optional)

```shell
[23:45:13] [pool-34-thread-1/WARN]: java.lang.NoSuchMethodError: 'boolean org.bukkit.entity.Player.canSee(org.bukkit.entity.Entity)'
[23:45:13] [pool-34-thread-1/WARN]: 	at dev.lone.cosmeticscore.main.i.c(SourceFile:658)
[23:45:13] [pool-34-thread-1/WARN]: 	at dev.lone.cosmeticscore.main.j.z(SourceFile:113)
[23:45:13] [pool-34-thread-1/WARN]: 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
[23:45:13] [pool-34-thread-1/WARN]: 	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
[23:45:13] [pool-34-thread-1/WARN]: 	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
[23:45:13] [pool-34-thread-1/WARN]: 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[23:45:13] [pool-34-thread-1/WARN]: 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[23:45:13] [pool-34-thread-1/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:833)
```
```


### CosmeticsCore config.yml

```shell
lang: en
extract_default_assets: true
cosmetics:
  max_equippable:
    body: 2
    balloon: 1
wardrobe_room:
  camera:
    hide_actionbar_texts: true
    rotation:
      enabled: true
    zoom:
      enabled: true
      min: 2
    fade_effect: true
    location:
      world: world
      x: -113.5
      y: 114.62000000476837
      z: -29.5
      yaw: -45.0
      pitch: 30.0
  mannequin:
    location:
      world: world
      x: -110.5
      y: 111.0
      z: -26.5
      yaw: 135.0
      pitch: 0.0
  teleport_area:
    enter:
      world: world
      pos1:
        x: -111.0
        y: 106.0
        z: -20.0
      pos2:
        x: -113.0
        y: 103.0
        z: -20.0
    exit_location:
      world: world
      x: -111.5
      y: 104.0
      z: -16.5
      yaw: 0.0
      pitch: 0.0
  gui:
    cursor_default: fermented_spider_eye:9000001
    cursor_rotate: fermented_spider_eye:9000002
    cursor_pointer: fermented_spider_eye:9000003
    dyeing_pointer: fermented_spider_eye:9100003
    tutorial_gui: fermented_spider_eye:9100004
    tutorial_gui_checked: fermented_spider_eye:9100005
    slots:
      colors:
        not_owned: 235,108,108
        owned: 56,67,100
        wearing: 57,70,195
        wearing_preview: 195,120,57
        max_amount_reached: 111,111,111
send_creative_hidden_hat_message: true
save:
  file:
    enabled: true
  mysql:
    enabled: false
    url: jdbc:mysql://my_url_database_69.com:3306/database_name
    username: username
    password: password
    table: cosmeticscore_saved
debug: false
```
```


### Problematic items yml configuration file (optional)

_No response_

### Other files, you can drag and drop them here to upload. (optional)

_No response_

### Screenshots/Videos (you can drag and drop files or paste links)


![image](https://user-images.githubusercontent.com/66962762/231610726-f9b85c83-fe0d-4ad7-ba56-a546638e5cf6.png)

**Comment by LoneDev6** [comment]
Duplicate of https://github.com/PluginBugs/Issues-CosmeticsCore/issues/109

**Comment by Lerphie** [comment]
> Duplicate of #109

When will you fix it?

**Comment by Lerphie** [comment]
I have a server opening tomorrow, I can't release it without this plugin

**Comment by LoneDev6** [comment]
It's fixed in the new update 1.2.2
